src/sensor/parser-ping.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include "parser.h"
+#include "ping-parser.h"
+
+/**
+ * @brief The PingParserExt class wraps the PingParser class from the ping-protocol submodule
+ * and Extends it with signalling in order to subclass our Parser class
+ */
+class PingParserExt : public Parser
+{
+public:
+    /**
+     * @brief Any messages parsed must be shorter than the buffer length
+     */
+    PingParserExt() : _parser(2048) {}
```

Comment:
Take in mind that Ping360 `device_data` can use up to 2026 points (if the max profile length is 2000 points).
This is not a microcontroller, we can put 10240 here just to be safe, or if you want to be safer and avoid multiple page memories, you can use 4096.

Reply:
done